lighthouse-core/report/v2/report-styles.css

```diff
@@ -182,53 +181,54 @@ summary {
   position: absolute;
   left: 0;
   right: 0;
-  top: 0;
+  top: -3px;
   bottom: 0;
-  background-color: #000;
   border-radius: inherit;
+  font-size: 1.5em;
+  text-align: center;
 }
 
 .lh-score__value--binary {
   text-indent: -500px;
 }
 
+.lh-score__value--binary::after {
+  text-indent: 0;
+}
+
 /* No icon for audits with number scores. */
 .lh-score__value:not(.lh-score__value--binary)::after {
   content: none;
 }
 
 .lh-score__value--pass {
-  background: var(--pass-color);
+  color: var(--pass-color);
 }
 
 .lh-score__value--pass::after {
-  background: url('data:image/svg+xml;utf8,<svg width="12" height="12" viewBox="0 0 12 12" xmlns="http://www.w3.org/2000/svg"><title>pass</title><path d="M9.17 2.33L4.5 7 2.83 5.33 1.5 6.66l3 3 6-6z" fill="white" fill-rule="evenodd"/></svg>') no-repeat 50% 50%;
```

Comment:
Can we keep the svg icons? They look cleaner than the ascii. You can change the size and `fill` color as needed.